tests/data/author/ssp/sample_nist_catalog.json

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
The only comment I have here is can some of the existing catalogs / ssp's be leveraged? I'm reticent to add yet more test files which introduce the potential refactoring effort on a new release.

Reply:
Removed extra data and used existing